featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -917,11 +925,6 @@ def wrap(df):
     return wrap
 
 
-def set_default_column(frame, f):
```

Comment:
Just curious, why did we delete this function?

Reply:
@rwedge Since this was a change you made before I picked this up, do you recall your thinking here?

Reply:
@thehomebrewnerd @sbadithe this code snippet is no longer used multiple times and is only two lines so I put it back in the function that still uses it